### DISCUSSION TITLE
Inconsistent badge theme

### DISCUSSION THREAD

**Discussion by yoannfleurydev** [BearStudio/start-ui-web#300] [Q&A] [answered]
Hi @ivan-dalmet , quick question about the badge theme.

[Those lines were edited two years ago](https://github.com/BearStudio/start-ui-web/blame/master/src/theme/components/badge.ts), but I'm asking my question anyhow (I think I know the answer though).

Do you remember why you did put `textTransform: 'none'` to `md` and `lg` size while the other have not those rules? I think that because `md` and `lg` are tall enough to read them correctly, so we don't really need the default uppercase, but to be honest, I'm not a fan of breaking the consistency of the component's style when changing the size.

What do you think?

**Comment by ivan-dalmet** [comment, marked as answer]
Yes that was the idea. But I agree with you, we should revert this inconsistency.
